Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -177,6 +177,9 @@ open class Kommunicate: NSObject, Localizable {
                             kmAppSetting.updateAppsettings(chatWidgetResponse: appSetting.chatWidget)
                             KMAppUserDefaultHandler.shared.isCSATEnabled
                                 = appSetting.collectFeedback ?? false
+                            if let zendeskaccountKey = appSetting.chatWidget?.zendeskChatSdkKey {
+                                ALApplozicSettings.setZendeskSdkAccountKey(zendeskaccountKey)
```

Comment:
See if we need to handle the case where Zopim is disabled, but user hasn't logged out. Otherwise, we'll have to add a note in the docs that disabling Zopim won't take effect without user log out.

Reply:
we are not handling. since we are getting that zendesk account key in appsetting api. As of now we can mention this in our doc.In future we can do some work around for this.